Mail/Views/Thread List/ThreadListCell.swift

```diff
@@ -61,13 +60,19 @@ struct ThreadListCell: View {
         }
         .padding(.leading, multipleSelectionViewModel.isEnabled ? 8 : -4)
         .background(SelectionBackground(isSelected: isSelected, verticalPadding: 2))
+        .background(SelectionBackground(isSelected: selectedThreadBackground, verticalPadding: 0))
```

Comment:
This creates two views, maybe we can refactor with an enum:
SelectionBackgroundKind: none, multipleSelection, singleSelection